diff -ruN a/application/CMakeLists.txt b/application/CMakeLists.txt
--- a/application/CMakeLists.txt	2022-09-13 10:40:04.833734800 +0200
+++ b/application/CMakeLists.txt	2022-09-13 10:40:21.485734667 +0200
@@ -30,7 +30,7 @@
 target_link_libraries(f3d PUBLIC libf3d)
 
 if (UNIX AND NOT APPLE)
-  target_link_libraries(f3d PRIVATE stdc++fs)
+  target_link_libraries(f3d PRIVATE)
 endif ()
 
 set_target_properties(f3d PROPERTIES RUNTIME_OUTPUT_DIRECTORY "${CMAKE_BINARY_DIR}/bin")
diff -ruN a/library/testing/CMakeLists.txt b/library/testing/CMakeLists.txt
--- a/library/testing/CMakeLists.txt	2022-09-07 12:17:51.000000000 +0200
+++ b/library/testing/CMakeLists.txt	2022-09-13 10:41:05.713734312 +0200
@@ -113,7 +113,7 @@
 endif()
 
 if (UNIX AND NOT APPLE)
-  target_link_libraries(libf3dSDKTests stdc++fs)
+  target_link_libraries(libf3dSDKTests)
 endif ()
 
 # make sure the libf3d API is compatible with C++11
